hq/app/schedule/IamNotifications.scala

```diff
@@ -5,18 +5,29 @@ import model._
 import org.joda.time.DateTime
 import play.api.Logging
 import schedule.IamMessages._
+import schedule.IamTargetGroups.getNotificationTargetGroups
 import schedule.Notifier.notification
 import schedule.vulnerable.IamDeadline.{createDeadlineIfMissing, sortUsersIntoWarningOrFinalAlerts}
 
 object IamNotifications extends Logging {
 
-  def makeNotification(flaggedCredsToNotify: Map[AwsAccount, Seq[IAMAlertTargetGroup]]): List[IamNotification] = {
-    flaggedCredsToNotify.toList.flatMap { case (awsAccount, targetGroups) =>
-      if (targetGroups.isEmpty) {
+  def makeNotifications(flaggedCredsToNotify: Map[AwsAccount, Seq[VulnerableUser]]): List[IamNotification] = {
+    flaggedCredsToNotify.toList.flatMap { case (awsAccount, users) =>
+      if (users.isEmpty) {
         logger.info(s"found no vulnerable IAM users for ${awsAccount.name}. No notification required.")
         None
       } else {
-        targetGroups.map(tg => createWarningAndFinalNotification(tg, awsAccount, createIamAuditUsers(tg.users, awsAccount)))
+        // TODO: can we just apply getNotificationTargetGroups for all accounts?
```

Comment:
As discussed elsewhere, we think this could probably be consolidated, but we'll leave that for another PR